### PR TITLE
koord-scheduler: fix Reservation normalize score

### PR DESCRIPTION
**Pull request by eahydra**
Signed-off-by: Joseph <joseph.t.lee@outlook.com>

### Ⅰ. Describe what this PR does

1. add unit test for Reservation with `scheduling.koordinator.sh/reservation-order`
2. fix Reservation normalize score

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: pkg/scheduler/plugins/reservation/plugin.go [hunk 1]
```diff
@@ -310,10 +310,13 @@ func (p *Plugin) Score(ctx context.Context, cycleState *framework.CycleState, po
 }
 
 func (p *Plugin) ScoreExtensions() framework.ScoreExtensions {
-	return nil
+	return p
```
Comment (saintube) [inline pkg/scheduler/plugins/reservation/plugin.go:313]: /lgtm

**Review by saintube — APPROVED** [review]

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**